dp-bulletin-api.nomad

```diff
@@ -0,0 +1,147 @@
+job "dp-bulletin-api" {
+  datacenters = ["eu-west-1"]
+  region      = "eu"
+  type        = "service"
+
+  update {
+    stagger          = "60s"
+    min_healthy_time = "30s"
+    healthy_deadline = "2m"
+    max_parallel     = 1
+    auto_revert      = true
+  }
+
+  group "web" {
+    count = "{{WEB_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "web"
+    }
+
+    restart {
+      attempts = 3
+      delay    = "15s"
+      interval = "1m"
+      mode     = "delay"
+    }
+
+    task "dp-bulletin-api" {
+      driver = "docker"
+
+      artifact {
+        source = "s3::https://s3-eu-west-1.amazonaws.com/{{DEPLOYMENT_BUCKET}}/dp-bulletin-api/{{REVISION}}.tar.gz"
+      }
+
+      config {
+        command = "${NOMAD_TASK_DIR}/start-task"
+
+        args = ["./dp-bulletin-api"]
+
+        image = "{{ECR_URL}}:concourse-{{REVISION}}"
+
+        port_map {
+          http = 8080
+        }
+      }
+
+      service {
+        name = "dp-bulletin-api"
+        port = "http"
+        tags = ["web"]
+
+        check {
+          type     = "http"
+          path     = "/health"
+          interval = "10s"
+          timeout  = "2s"
+        }
+      }
+
+      resources {
+        cpu    = "{{WEB_RESOURCE_CPU}}"
+        memory = "{{WEB_RESOURCE_MEM}}"
+
+        network {
+          port "http" {}
+        }
+      }
+
+      template {
+        source      = "${NOMAD_TASK_DIR}/vars-template"
+        destination = "${NOMAD_TASK_DIR}/vars"
+      }
+
+      vault {
+        policies = ["dp-bulletin-api-web"]
+      }
+    }
+  }
+
+  group "publishing" {
+    count = "{{PUBLISHING_TASK_COUNT}}"
+
+    constraint {
+      attribute = "${node.class}"
+      value     = "publishing"
+    }
+
+    restart {
+      attempts = 3
+      delay    = "15s"
+      interval = "1m"
+      mode     = "delay"
+    }
+
+    task "dp-bulletin-api-publishing" {
+      driver = "docker"
+
+      artifact {
+        source = "s3::https://s3-eu-west-1.amazonaws.com/{{DEPLOYMENT_BUCKET}}/dp-bulletin-api/{{REVISION}}.tar.gz"
+      }
+
+      config {
+        command = "${NOMAD_TASK_DIR}/start-task"
+
+        args = ["./dp-bulletin-api"]
+
+        image = "{{ECR_URL}}:concourse-{{REVISION}}"
+
+        port_map {
+          http = 24200
```

Comment:
@janderson2 Similar to above, could you please clarify this one?  (`"${NOMAD_PORT_http}"`)…
https://github.com/ONSdigital/dp-search-query/blob/develop/dp-search-query.nomad#L110
https://github.com/ONSdigital/dp-dataset-api/blob/develop/dp-dataset-api.nomad#L110
https://github.com/ONSdigital/dp-search-api/blob/develop/dp-search-api.nomad#L110

Whereas other(s) have a specific port number (eg.  `http = 24100`)…
https://github.com/ONSdigital/dp-frontend-cookie-controller/blob/develop/dp-frontend-cookie-controller.nomad#L111

Reply:
Because we pass in the `BIND_ADDR` as an env var using `:$NOMAD_PORT_http` in the app configs, the `port_map` is not actually needed and can be removed.